src/PublicAPI.targets

```diff
@@ -25,4 +37,11 @@
     <AdditionalFiles Include="@(PublicAPIFiles)" />
   </ItemGroup>
 
+  <Target Name="_GeneratePublicApiFiles" AfterTargets="Build" Condition="'$(TargetFramework)' != '' and '$(PublicApiType)' == 'Generate'">
+    <GeneratePublicApiFiles
+      Assembly="$(TargetDir)$(AssemblyName).dll"
+      Files="@(AdditionalFiles)"
+      ReferenceSearchPaths="@(ReferencePath)" />
+  </Target>
```

Comment:
Can you add `Inputs`/`Outputs`, so it doesn't run on every incremental build for every project?

Is this right?

```xml
Inputs="$(TargetDir)$(AssemblyName).dll"
Outputs="@(PublicAPIFiles)"
```